SmartDeviceLink/SDLScreenManager.h

```diff
@@ -28,8 +34,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 typedef void(^SDLScreenManagerUpdateCompletionHandler)(NSError *__nullable error);
 
+/**
+ Return an error with userinfo [key: SDLChoiceCell, value: NSError] if choices failed to upload
+
+ @param error The error if one occurred
+ */
+typedef void(^SDLPreloadChoiceCompletionHandler)(NSError *error);
```

Comment:
The returned error in the `SDLPreloadChoiceCompletionHandler` should be `nullable`.
Also, there appears to be a duplicate of `SDLPreloadChoiceCompletionHandler` in the _SDLChoiceSetManager_ class.

Reply:
Fixed